src/utils/createRandomRGB.js

```diff
@@ -0,0 +1,10 @@
+const createRandomRGB = () => {
+	const R = Math.floor(Math.random() * 255).toString();
+	const G = Math.floor(Math.random() * 255).toString();
+	const B = Math.floor(Math.random() * 255).toString();
```

Comment:
You don't need to use `.toString()` here and you can use `createRandomInt`:
```js
const R = createRandomInt(255);
const G = createRandomInt(255);
const B = createRandomInt(255);
```
or
```js
const createRandomRGB = () =>
  `rgb(${createRandomInt(255)}, ${createRandomInt(255)}, ${createRandomInt(255)})`;
```